report.php

```diff
@@ -19,6 +19,7 @@
     header('Location: index.php');
     die();
   }
+  /*
```

Comment:
No comment was left here. What was the reason for commenting out the PHP code?

Reply:
For one of the tasks I am handling this sprint, I am creating the front-end of generating reports but the old odhs page was failing due to sql problems. Since I did not need that yet, I commented it out but also commented out the header bar functionality by accident